roles/sft-server/vars/main.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: considered to be constants due to high precedence
```

Comment:
Please move all of these variables to `defaults`, otherwise they are impossible to override. A variable that cannot be overridden is rather useless, no?